### PR TITLE
Small example that loads a Carto Torque tile and renders using deck.gl scatterplot

### PR DESCRIPTION
**Pull request by ibgreen**
Based on tile description and URL provided in https://gist.github.com/javisantana/68e50e70bf5463dbcaaf205063fcef00

screen shot 2017-05-19 at 9 13 44 am

Currently shows all the points in the tile at the same time. Next step - implement animation.

@shaojingli @philogb @javisantana

### CODE REVIEW

**Comment by ericsoco** [comment]
Heyyy, look at that 👍

**Comment by shaojingli** [comment]
Great!

**Comment by shaojingli** [comment]
may-19-2017 14-17-22

Animating the data using a very naive time loop

**Comment by ericsoco** [comment]
FWIW I believe Torque's past examples show "trails", in which older timesteps stay on-screen for a while with reduced opacity. Would help sell the PoC to add that :)

**Comment by shaojingli** [comment]
@ericsoco yeah... that's next step.. we will need a new layer that draw a quad blended from two textures from two framebuffers.

**Comment by shaojingli** [comment]
Added some alpha blending for fade in and fade out

may-19-2017 15-58-07

File: examples/torque/time-sliced-scatterplot-layer/time-sliced-scatterplot-layer-vertex.glsl.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  // Find the center of the point and add the current vertex
+  vec3 center = project_position(instancePositions);
+  vec3 vertex = positions * outerRadiusPixels;
+  gl_Position = project_to_clipspace(vec4(center + vertex, 1.0));
+
+  // Apply opacity to instance color, or return instance picking color
+  vec4 color = vec4(instanceColors.rgb, instanceColors.a * opacity) / 255.;
+
+  color.a = 1.0 - abs(currentTime - time) * fadeFactor;
```
Comment (ericsoco) [inline examples/torque/time-sliced-scatterplot-layer/time-sliced-scatterplot-layer-vertex.glsl.js]: To make the current time layer stand out a bit more against the fading timesteps, we might want to make this more like `0.75 -` or event `0.5 -`.

**Comment by shaojingli** [comment]
Updated the torque example to correctly interpret the data according to https://gist.github.com/javisantana/68e50e70bf5463dbcaaf205063fcef00

Here is what I got

ship

Tile loading is still static though.

**Comment by ibgreen** [comment]
Closing as the data sources are no longer valid and another example has been landed on master.

**Pull request closed without merging**